src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowController.cs

```diff
@@ -90,7 +90,7 @@ public void RunPowerShellTest(XunitTracingInterceptor logger, params string[] sc
                     "ScenarioTests\\Common.ps1",
                     "ScenarioTests\\" + callingClassName + ".ps1",
                     _helper.RMProfileModule,
-                    _helper.GetRMModulePath(@"AzureRM.LogicApp.psd1"),
+                    _helper.GetRMModulePath(@"Az.LogicApp.psd1"),
```

Comment:
I don't think we support this yet. AFAIK, it still needs to be `"AzureRM.LogicApp.psd1"` for it to resolve correctly in `GetRMModulePath`.

Reply:
Discussed offline. Should work with `"Az.LogicApp.psd1"`.